test/e2e-common.sh

```diff
@@ -187,6 +187,9 @@ function install_knative_eventing() {
     UNINSTALL_LIST+=( "${EVENTING_RELEASE_YAML}" )
   fi
 
+  # Workaround for https://github.com/knative/eventing/issues/8161
+  kubectl label namespace "${SYSTEM_NAMESPACE}" bindings.knative.dev/exclude=true --overwrite
```

Comment:
Can we have some tracker to remove this workaround and/or note that this should be cleaned up on #8161 ?